e2e/utils/test_cases.js

```diff
@@ -83,13 +85,47 @@ function saveToEndpoint(url, data) {
     });
 }
 
+async function getZEPHYRFolderID() {
+    const {
+        TYPE,
+        ZEPHYR_FOLDER_ID,
+    } = process.env;
+    if (TYPE === 'MASTER') {
+        return ZEPHYR_FOLDER_ID;
+    }
+    const platform = os.platform();
+
+    // Define Zephyr folder IDs for different run types and platforms.
+    // For PR we dont generate reports.
+    // Post Merge to master branch, default folderID will be used.
+    const folderIDs = {
+        RELEASE: {
+            darwin: 12358650,
+            win32: 12358651,
+            linux: 12358649,
+            default: ZEPHYR_FOLDER_ID,
+        },
+        NIGHTLY: {
+            darwin: 12363687,
+            win32: 12363690,
+            linux: 12363689,
+            default: ZEPHYR_FOLDER_ID,
+        },
+    };
```

Comment:
I feel like this should not be hardcoded here, and should be declared depending on OS matrix strategy at workflow file.

Reply:
@saturninoabril  sure Saturn, that would be a good place to keept this